content/en/resources/_index.md

```diff
@@ -36,6 +36,7 @@ type: docs
 
 *(latest to oldest)*
 
+- [Mirroring environments with gitops and Kyverno](https://gimlet.io/blog/mirroring-environments-with-gitops-and-kyverno/)
```

Comment:
Please add the date the post was authored to be consistent with the other blog post listings.

Reply:
It's added now.